rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,59 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 
 When a recording state changes, a toast notification appears indicating current state. The notification stays for two seconds and disappears automatically.
 
+== Image-Based Mouse Clicks
+
+Use image-based mouse clicks to record mouse clicks on a UI element that otherwise cannot be recorded properly. The feature guides you through the mouse click process. An image on the screen and mouse clicks must be captured, and the mouse clicks are recorded relative to the captured image.
+
+Use this feature at any point during a normal recording session. The feature also checks your display settings automatically while recording to ensure the feature records correctly.
+
+This feature will result in _Search Pattern_ and _Mouse Action_ Action Steps when the workflow is generated in RPA Builder from the recording.
+
+This feature is available for _Search Pattern_ and _Mouse Action_ Action Steps.
+
+When using this feature:
+
+* The Recording Toolbar is inactive.
+* Keyboard events are not recorded.
+
+=== Recommended Display Settings
+
+Apply the following displays settings to capture image-based mouse clicks:
+
+* Single monitor. Multiple monitors are supported, but the capturing will be limited to the primary display only for Bot tasks and to the web page for Web tasks.
```

Comment:
```suggestion
* Single monitor. 
+
Multiple monitors are supported, but the capturing will be limited to the primary display only for Bot tasks and to the web page for Web tasks.
```

Reply:
Kim wanted us to put extra bullet text on a new line after a horizontal space to make the text more readable. (I marked this for the first bullet item.)